content/kb/analysis-filesystem-corrupted-issues-due-to-error-on-cow-while-rebuilding-replicas.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Analysis: Potential Data/Filesystem Corruption"
+author: Derek Su and Phan Le
+draft: false
+date: 2022-09-29
+categories:
+- "data-integrity"
+---
+
```

Comment:
This is good from a technical details points, let's add a small summary above the affected versions.

Makings sure that the user knows that this isn't very likely to happen to them (needs to be rebuilding, non 4k (default) block size) and if so they would have catched it most likely after the next mount.